myriad-scheduler/src/main/java/org/apache/myriad/scheduler/ServiceTaskFactoryImpl.java

```
@@ -80,174 +68,33 @@ public TaskInfo createTask(Offer offer, FrameworkID frameworkId, TaskID taskId,
-        yarnEnvB.setName(envEntry.getKey()).setValue(envEntry.getValue());
-        yarnHomeB.addVariables(yarnEnvB.build());
-      }
-      commandInfo.mergeEnvironment(yarnHomeB.build());
-    }
-
-    if (myriadExecutorConfiguration.getNodeManagerUri().isPresent()) {
-      //Both FrameworkUser and FrameworkSuperuser to get all of the directory permissions correct.
-      if (!minimumUserSet(cfg)) {
-        throw new RuntimeException("Trying to use remote distribution, but frameworkUser" + "and/or frameworkSuperUser not set!");
-      }
-
-      LOGGER.info("Using remote distribution");
-      String clGeneratedCommand = clGenerator.generateCommandLine(profile, null);
-
-      String nmURIString = myriadExecutorConfiguration.getNodeManagerUri().get();
-
-      //Concatenate all the subcommands
-      String cmd = clGeneratedCommand + " " + executorCmd;
-
-      //get the nodemanagerURI
-      //We're going to extract ourselves, so setExtract is false
-      LOGGER.info("Getting Hadoop distribution from:" + nmURIString);
-      URI nmUri = URI.newBuilder().setValue(nmURIString).setExtract(false).build();
-
-      //get configs directly from resource manager
-      String configUrlString = clGenerator.getConfigurationUrl();
-      LOGGER.info("Getting config from:" + configUrlString);
-      URI configUri = URI.newBuilder().setValue(configUrlString).build();
-
-      LOGGER.info("Slave will execute command:" + cmd);
-      commandInfo.addUris(nmUri).addUris(configUri).setValue("echo \"" + cmd + "\";" + cmd);
-      commandInfo.setUser(cfg.getFrameworkSuperUser().get());
-
-    } else {
-      commandInfo.setValue(executorCmd);
-    }
-    return commandInfo.build();
-  }
-
-  private Boolean minimumUserSet(MyriadConfiguration conf) {
-    return (cfg.getFrameworkUser().isPresent() || cfg.getFrameworkSuperUser().isPresent());
-  }
-  
   @Override
   public ExecutorInfo getExecutorInfoForSlave(FrameworkID frameworkId, Offer offer, CommandInfo commandInfo) {
```

Perhaps, the method isn't actually used so no NPE risk (for now).  It's defined as the original interface required it and I make the method abstract when I change to a class.  But I left the method as it was.

Maybe throw an unsupported method exception?